examples_tests/61.BlitFilterTest/blit/formats/r11f_g11f_b10f.glsl

```diff
@@ -0,0 +1,78 @@
+#ifndef _NBL_GLSL_BLIT_R11FG11FB10F_INCLUDED_
+#define _NBL_GLSL_BLIT_R11FG11FB10F_INCLUDED_
+
+uint to11bitFloat(in float _f32)
+{
+	const uint f32 = floatBitsToUint(_f32);
+
+	if ((f32 & 0x80000000u) != 0u)
+		return 0u;
+
+	const uint f11MantissaMask = 0x3fu;
+	const uint f11ExpMask = 0x1fu << 6;
+
+	const int exp = int(((f32 >> 23) & 0xffu) - 127);
+	const uint mantissa = f32 & 0x7fffffu;
+
+	uint f11 = 0u;
+	if (exp == 128) // inf / NaN
+	{
+		f11 = f11ExpMask;
+		if (mantissa != 0u)
+			f11 |= (mantissa & f11MantissaMask);
+	}
+	else if (exp > 15) // overflow converts to infinity
+		f11 = f11ExpMask;
+	else if (exp > -15)
+	{
+		const int e = exp + 15;
+		const uint m = mantissa >> (23 - 6);
+		f11 = (e << 6) | m;
+	}
+
+	return f11;
+}
+
+uint to10bitFloat(in float _f32)
+{
+	const uint f32 = floatBitsToUint(_f32);
+
+	if ((f32 & 0x80000000u) != 0u) // negative numbers converts to 0 (represented by all zeroes in 10bit format)
+		return 0;
+
+	const uint f10MantissaMask = 0x1fu;
+	const uint f10ExpMask = 0x1fu << 5;
+
+	const int exp = int(((f32 >> 23) & 0xffu) - 127);
+	const uint mantissa = f32 & 0x7fffffu;
+
+	uint f10 = 0u;
+	if (exp == 128) // inf / NaN
+	{
+		f10 = f10ExpMask;
+		if (mantissa != 0u)
+			f10 |= (mantissa & f10MantissaMask);
+	}
+	else if (exp > 15) // overflow, converts to infinity
+		f10 = f10ExpMask;
+	else if (exp > -15)
+	{
+		const int e = exp + 15;
+		const uint m = mantissa >> (23 - 5);
+		f10 = (e << 5) | m;
+	}
+
+	return f10;
+}
```

Comment:
maybe make a generic `nbl_glsl_encode_ufloat(in float f32, in uint exponentBits, in uint mantissaBits)` the two paths look very similar

Reply:
you want to copy and conform to `nbl_glsl_impl_sharedExponentEncodeCommon` as much as possible (maybe even factor out common things)